builder/vmware/common/driver.go

```diff
@@ -256,38 +264,31 @@ func runAndLog(cmd *exec.Cmd) (string, string, error) {
 	return returnStdout, returnStderr, err
 }
 
-// Still used for Workstation and Player until conversion.
-func normalizeVersion(version string) (string, error) {
-	i, err := strconv.Atoi(version)
+// compareVersions compares two version strings and returns an error if the
+// found version is less than the required version.
+func compareVersions(versionFound, versionRequired, product string) error {
+	vFound, err := version.NewVersion(versionFound)
 	if err != nil {
-		return "", fmt.Errorf("returned a non-integer version %q: %s", version, err)
+		return fmt.Errorf("invalid version found: %w", err)
 	}
 
-	return fmt.Sprintf("%02d", i), nil
-}
-
-// Still used for Workstation and Player until conversion.
-func compareVersions(versionFound string, versionWanted string, product string) error {
-	found, err := normalizeVersion(versionFound)
+	vRequired, err := version.NewVersion(versionRequired)
```

Comment:
Regarding this, I believe the version here is one of the versions from the `const` block right? The `err` could be ignored here, provided we have a semi-static validation that this works with `version`, you can probably add a var decl block with several `version.Must(version.NewVersion([...]))` for all the statically-known versions meant to be used here.

Reply:
I've made some updates but pending tests.